unit_testing/create_test_data.py

```diff
@@ -16,7 +16,7 @@
 
 def dummy_blob(size_arr=(9, 9, 9), pixdim=(1, 1, 1), coordvox=None, debug=False):
     # nx, ny, nz = size_arr
-    data = np.zeros(size_arr)
+    data = np.zeros(size_arr, dtype=np.uint8)
```

Comment:
This might hide bugs. Is real data necessarily uint8? I would make this a parameter.

Reply:
fixed 0fa3111388a23fee28ac82fc55e223cd105dc17e